CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.0.0 (21.8.2018)
```

Comment:
Leave this out for now, maintainers will update the changelog when a new release is made